integrations/airflow/openlineage/facet.py

```diff
@@ -0,0 +1,119 @@
+import os
+from typing import Dict, List
+
+import attr
+
+from openlineage.constants import DEFAULT_PRODUCER
+
+
+PRODUCER = os.getenv("MARQUEZ_PRODUCER", DEFAULT_PRODUCER)
+SCHEMA_URI = "https://raw.githubusercontent.com/OpenLineage/OpenLineage/main/spec/OpenLineage.json"
+
+
+@attr.s
+class BaseFacet:
```

Comment:
Same here- all these facets are general and I think they're better off in a separate OpenLineage client module